packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -646,9 +648,13 @@ export class ConnectionManager implements IConnectionManager {
             clientId: connection.clientId,
             mode: connection.mode,
         };
+
         if (connection.relayServiceAgent !== undefined) {
             this._connectionVerboseProps.relayServiceAgent = connection.relayServiceAgent;
+            this._connectionProps.relayServiceAgent = connection.relayServiceAgent;
         }
+        this._connectionProps.connectionMode = connection.mode;
```

Comment:
I'd do two minor additions:
- reset _connectionProps on line 651 to be empty, just in case
- add socketDocumentId here to this._connectionProps as well - it's in line with the how we log other properties (i.e. we will log it on disconnect as well, and it will represent state on disconnecting connection).